docs/sources/dashboards/time-range-controls.md

```diff
@@ -0,0 +1,116 @@
+name = "Time range controls"
+parent = "dashboards"
+weight = 7
++++
+
+# Time range controls
+
+Grafana provides several ways to manage the time ranges of the data being visualized, both at the dashboard level and the panel level. 
+
+This page describes supported time units and relative ranges, the common time controls, dashboard-wide time settings, and panel-specific time settings.
+
+## Time units and relative ranges
+
+The following time units are supported: `s (seconds)`, `m (minutes)`, `h (hours)`, `d (days)`, `w (weeks)`, `M (months)`, and `y (years)`. 
+
+The minus operator allows you to step back in time, relative to now. If you wish to display the full period of the unit (day, week, month, etc...), append `/$unit` to the end.
+
+Here are some examples:
+
+Example relative range | From: | To:
+-------------- | ----- | ---
+Last 5 minutes | `now-5m` | `now`
+The day so far | `now/d` | `now`
+This week | `now/w` | `now/w`
+Week to date | `now/w` | `now`
+Previous Month | `now-1M/M` | `now-1M/M`
+
+## Common time range controls
+
+The dashboard and panel time controls have a common user interface (UI).
+
+<img class="no-shadow" src="/img/docs/time-range-controls/common-time-controls-7-0.png" max-width="700px">
+
+The options are defined below.
+
+### Current time range
+
+The current time range, also called the _time picker_, shows the time range currently displayed in the dashboard or panel you are viewing.
+
+Hover your curser over the field to see the exact time stamps in the range and their source (such as the local browser).
+
+<img class="no-shadow" src="/img/docs/time-range-controls/time-picker-7-0.png" max-width="300px">
+
+To change the current time range, click the down arrow next to the current time range. You can change the current time using a _relative time range_, such as the last 15 minutes, or an _absolute time range_, such as `now-45h`.
+
+<img class="no-shadow" src="/img/docs/time-range-controls/change-current-time-range-7-0.png" max-width="900px">
+
+### Relative time range
+
+Select the relative time range from the **Relative time ranges** list. Some examples of time ranges are:
+
+- Last 30 minutes
+- Last 12 hours
+- Last 7 days
+- Last 2 years
+- Yesterday
+- Day before yesterday
+- This day last week
+- Today so far
+- This week so far
+- This month so far
+
+### Absolute time range
+
+Set an absolute time range one of two ways:
+
+- Type values into the **From** and **To** fields. You can type exact time values or relative values, such as `now-24h`, and then click **Apply time range**.
+- Click in the **From** or **To** field. Grafana displays a calender. Click the day or days you want to use as the current time range and then click **Apply time range**.
+
+This section also displays recently used absolute ranges.
+
+### Zoom out (Cmd+Z or Ctrl+Z)
+
+Click the **Zoom out** icon to view a larger time range in the dashboard or panel visualization.
+
+### Zoom in (not shown in UI)
```

Comment:
Don't really understand this. Are you referring to zooming in to a time range in the graph panel?

Reply:
This is literally the answer I got when I asked "how do you zoom in".

Reply:
Updated content to clarify about the graph panel